software/01-OpenBCI_SDK.md

```diff
@@ -177,15 +199,61 @@ Use 8 channels only. If the Daisy Module is attached, it will be unattached, and
 
 **C**
 
-Use 16 channels. 
+Use 16 channels.
 
 *Note: On reset, the OpenBCI 32bit board will 'sniff' for the Daisy Module, and if it is present, it will default to 16 channel capability.*
 
-##Unused ASCII Characters
-These are currently unused characters across the OpenBCI platforms:
+#Firmware v2.0.0 New Commands
+
+##Time Stamping
+
+**<**  
+Start time stamping and resynchronize command. The Host radio will send a **,** on send of the **<** back to the driver. Since the Host cannot send packets to the Device ad hoc, it may be helpful to know when the Host was actually able to send the command. If the Board is not streaming, then expect a response of `Time stamp ON$$$`; however if the board is streaming, then you will get a response in a different **stop byte** as described in the next software document titled _OpenBCI Streaming Data Format_.
```

Comment:
Start time stamping and resynchronize command. When the Driver sends a **<**, the Host radio will respond with a **,**. Since the Host cannot send packets to the Device ad hoc, it may be helpful to know when the Host was actually able to send the command. If the Board is not streaming, then expect a response of `Time stamp ON$$$`; however if the board is streaming, then you will get a response in the data stream when the Driver gets a data packet with a different **stop byte** as described in the next software document titled _OpenBCI Streaming Data Format_.